flink-table/flink-table-common/src/main/java/org/apache/flink/table/connector/RowLevelModificationScanContext.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.connector;
+
+import org.apache.flink.annotation.PublicEvolving;
+import org.apache.flink.table.connector.source.abilities.SupportsRowLevelModificationScan;
+
+/**
+ * The context for scan the table to do updated/delete, designed to provide some necessary
```

Comment:
'This context is intended to provide the relevant table scan information needed by the sink to perform a row-level update/delete. It will be generated by a table source that implements {@link SupportsRowLevelModificationScan} and then passed to a sink that implements {@link SupportsRowLevelUpdate} or {@link SupportsRowLevelDelete} for executing UPDATE/DELETE statements during the compilation phase.
This mechanism enables coordination between the source and sink for the corresponding table updates/deletes. Connectors can implement this interface to provide custom information.'

Reply:
For the sendond paragrah, I change it to 
`
This mechanism enables the coordination between the table sources and the table sink which is to be updated/deleted.
`